app/notify.js

```diff
@@ -27,9 +27,13 @@ app.on('ready', () => {
   });
 });
 
-function notify(title, body) {
+function notify(title, body, error) {
```

Comment:
I think it would be a good idea to do {error} as an object on the end, so if we need to send more info to notify in the future we don't need to have another positional argument 👍

Reply:
It's funny you mention that, because that was actually my first thought!